api/tests/test_building_api.py

```diff
@@ -31,6 +31,37 @@ def test_can_post_office_block(self, mock_verify_token):
         self.assertIn("name", response.data.keys())
         self.assertEqual(response.status_code, 201)
 
+    @patch('api.authentication.auth.verify_id_token')
+    def test_get_office_blocks_in_a_center_by_super_admin(self, mock_verify_token):
+        mock_verify_token.return_value = {'email': self.admin_user.email}
+        url = "/api/v1/andela-centres/{}/office_blocks/".format(self.centre.id)
```

Comment:
This doesn't show that a super_admin could get office_blocks from other centers other than theirs. You should test that a super admin in self.centre can get office blocks in self.other_centre.

Reply:
done